adobe_vipm/adobe/mixins/subscription.py

```diff
@@ -43,34 +40,54 @@ def get_subscription(
         return response.json()
 
     @wrap_http_error
-    def get_subscriptions(
-        self,
-        authorization_id: str,
-        customer_id: str,
-    ) -> dict:
+    def get_subscriptions(self, authorization_id: str, customer_id: str) -> dict:
         """
         Retrieve all the subscriptions of the given customer.
 
         Args:
-            authorization_id: Id of the authorization to use.
-            customer_id: Identifier of the customer to which the subscriptions belongs to.
+            authorization_id: ID of the authorization to use.
+            customer_id: Identifier of the customer to which the subscriptions belong to.
 
         Returns:
-            dict: The retrieved subscriptions.
+            The retrieved subscriptions.
         """
         authorization = self._config.get_authorization(authorization_id)
-        headers = self._get_headers(authorization)
         response = requests.get(
-            urljoin(
-                self._config.api_base_url,
-                f"/v3/customers/{customer_id}/subscriptions",
-            ),
-            headers=headers,
+            urljoin(self._config.api_base_url, f"/v3/customers/{customer_id}/subscriptions"),
+            headers=self._get_headers(authorization),
             timeout=self._TIMEOUT,
         )
         response.raise_for_status()
         return response.json()
 
+    # TODO: Merge this method with get_subscriptions in a separate PR. Created to prevent
+    # breaking existing code.
+    @wrap_http_error
+    def get_subscriptions_by_deployment(
```

Comment:
I think we also should merge it with the

https://github.com/softwareone-platform/swo-adobe-vipm-extension/blob/main/adobe_vipm/flows/global_customer.py#L55